crates/apollo-compiler/src/database/sources.rs

```diff
@@ -62,7 +105,7 @@ impl Source {
     }
 }
 
-#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash, Ord, PartialOrd)]
```

Comment:
Instead of ordering and sorting, what if `NEXT` started at 1 and the built-in source always had ID zero?

Reply:
we could do that too, instead of `Ord` and `PartialOrd` we'd have a `impl From<u64> for FileId` instead to cast the always-zero ID, which seems like a smaller API change anyways?

Reply:
ah, but it doesn't matter as the Debug output which we write to the test output still spits the file paths out in an indeterminate order. i think easier to have `Ord` and `PartialOrd` then.